pkgs/build-support/libredirect/libredirect.c

```diff
@@ -139,7 +139,7 @@ WRAPPER_DEF(open64)
 
 WRAPPER(int, openat)(int dirfd, const char * path, int flags, ...)
 {
-    int (*openat_real) (int, const char *, int, mode_t) = LOOKUP_REAL(openat);
+    int (*openat_real) (int, const char *, int, ...) = LOOKUP_REAL(openat);
```

Comment:
Eh, I expect someone was looking at Linux docs instead of POSIX.  I'm prone to doing that mistake, too.